### ISSUE TITLE
[MU4 Issue] MIDI keyboard: No live playback of instruments or input of notes (regression)

### ISSUE THREAD

**Issue by rgreen5** [musescore/MuseScore#14534]
**Describe the bug**

This a recent bug. Possibly due to the attempted fix here: #11234.

Now, no input of notes is possible at all with a MIDI keyboard (Akai LPK"5); this was working before the "fix", and works OK with MS3 on the same system. Tested using the JAZZ COMBO template.

There is no live playback of instruments either, which suggests that the previously-mentioned fix (#11234) hasn't worked. In fact it also seems to have caused an additional problem with input from the on-screen piano – which is reported as a separate bug.

**Expected behavior**

Live playback of an instrument should be possible by selecting a measure (not a note or rest) in a staff and then playing an attached MIDI keyboard. This was how it worked in MS2. Input of notes should be possible in Note input mode. The foregoing should apply to the on-screen piano keyboard as well.

**Platform information**

OS: Linux Mint 20.1, Arch.: x86_64, MuseScore version (64-bit): 4.0.0-223190408, revision: github-musescore-musescore-3558138

**Comment by Eism** [comment]
@rgreen5 
Yes, sound preview didn't work for Piano keyboard. Fixed, see https://github.com/musescore/MuseScore/pull/14538 
But the preview works without my changes for input from a midi keyboard. Tested on Linux and macOS

Correct me if I'm wrong.

**Comment by rgreen5** [comment]
OS: Linux Mint 20.1, Arch.: x86_64, MuseScore version (64-bit): 4.0.0-223190408, revision: github-musescore-musescore-3558138 / Akai LPK25 USB MIDI keyboard

I'm getting absolutely nothing; no preview, no note input. I'm using the same ALSA sound system as always. It works with MS3.

**Comment by DmitryArefiev** [comment]
Can you try Revert factory settings? 
If not, Does it appear in Preferences>I/O>MIDI input device?
Does it appear in MS3 (in Preferences>I/O>MIDI input device)?

**Comment by rgreen5** [comment]
Yes, Revert to factory settings did the trick. A-OK now.

**Comment by DmitryArefiev** [comment]
> OS: Linux Mint 20.1, Arch.: x86_64, MuseScore version (64-bit): 4.0.0-223210406, revision: github-musescore-musescore-089ce9f
> 
> Yes, Revert to factory settings did the trick. A-OK now.

Cool! Please reopen if the issue happens again
